.github/workflows/deploy.yml

```diff
@@ -0,0 +1,63 @@
+run-name: Deploy version ${{ github.event.release.tag_name }} to TT02 and Production
+on:
+  release:
+    types: [released]
+          
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: altinn/altinn-auth-audit-log:${{ github.event.release.tag_name }}
+  CONTAINER_APP_CONTAINER_NAME: auditlog
+  
+jobs:
+  deploy-tt02:
+    environment: TT02
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      packages: read
+    steps:
+    - name: Azure Login
+      uses: azure/login@v1
+      with:
+        client-id: ${{ secrets.AZURE_CLIENT_ID }}
+        tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+        subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+    - name: Deploy to containerapp
+      uses: azure/CLI@v1
+      with:
+        inlineScript: >
+          az config set extension.use_dynamic_install=yes_without_prompt
+          
+          az containerapp update \
+            --name ${{ vars.CONTAINER_APP_NAME }} \
```

Comment:
It's generally recommended to not use string interpolation in scripts in github actions. Instead you should pass in environment variables and refer to them in the script. This has several advantages, several of which are safety related.